inc/class-markdownparser.php

```diff
@@ -63,4 +63,27 @@ protected function inlineLink( $data ) {
 
 		return $result;
 	}
+
+	/**
+	 * Parse a header block
+	 *
+	 * Override the blockHeader method to add anchor links.
+	 *
+	 * @param array $data
+	 * @return array
+	 */
+	protected function blockHeader( $data ) {
+		$block = parent::blockHeader( $data );
+		$id = str_replace( ' ', '-', strtolower( strip_tags( $block['element']['handler']['argument'] ) ) );
```

Comment:
Can we use a WP function instead of `strip_tags` here?

Reply:
We probably could, but this was just a visual fix more than a security related thing, so I think the PHP internal function is probably fine.

Reply:
Yeah, it's more that I think we should be consistent. `sanitize_title_with_dashes` is possibly a better ID generator here?